tests/phpunit/tests/functions/sendFrameOptionsHeader.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Tests for the send_frame_options_header function.
+ * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
+ *
+ * @group functions
+ * @group xdebug
+ *
+ * @covers ::send_frame_options_header
+ */
+class Tests_Functions_SendFrameOptionsHeader extends WP_UnitTestCase {
+
+
+	/**
+	 * Just test for function
+	 *
+	 * @ticket 59851
+	 *
+	 * @requires function xdebug_get_headers
+	 */
+	public function test_send_frame_options_header() {
+		$this->assertTrue( function_exists( 'send_frame_options_header' ) );
```

Comment:
Can you elaborate on this assertion? When should this function not exist?

Reply:
I will remove it 
lets over from the old version